src/scripts/get-all-files.test.ts

```diff
@@ -0,0 +1,25 @@
+import getAllFiles, { ReaddirReturn } from "./get-all-files";
+
+jest.mock("readdirp", () => ({
+  promise(root: string): ReaddirReturn {
+    return Promise.resolve([
+      {
+        path: "./example.json",
+        fullPath: `${root}/example.json`,
+        basename: "example.json"
+      }
+    ]);
+  }
+}));
+
+describe("get all files in folder", () => {
+  it("should find the files", async () => {
+    expect(await getAllFiles("/test")).toEqual([
+      {
+        basename: "example.json",
```

Comment:
This test feels a bit off, it tests that the mocked json, equals the mocked json.
the reason it is this way, is the module its testing, wraps a single line, external function.
This may not be needed, this function may be more useful folded into a larger abstraction.